tools/runner

```diff
@@ -96,16 +96,17 @@ try:
             param_name = param.group(1).lower()
             param_value = param.group(2)
 
-            if param_name not in req_test_params:
+            if (param_name not in req_test_params):
```

Comment:
I don't think we need extra parentheses here

Reply:
yup force of habit, I always use parentheses to make my intention clear, but if you don't use it unless necessary i can remove it no problem.